config.schema.yml

```diff
@@ -122,11 +122,19 @@ properties:
                                         type: "array"
                                         items:
                                             type: "string"
-                                    
+
                     enableReload:
                         type: "boolean"
                     roomLimit:
                         type: "number"
+                    secret:
+                        type: "string"
+                    apiPrefix:
+                        type: "string"
+                    openIdDisallowedIpRanges:
```

Comment:
You might also want an openIdOverrides similar to hookshot (https://matrix-org.github.io/matrix-hookshot/latest/advanced/widgets.html#configuration) which would be helpful when local development happens (and your Synapse isn't on HTTPS).